src/main/java/at/aau/serg/websocketdemoserver/msg/GameMessageImpl.java

```diff
@@ -0,0 +1,19 @@
+package at.aau.serg.websocketdemoserver.msg;
+
+import at.aau.serg.websocketdemoserver.model.game.Field;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class GameMessageImpl implements BaseMessageImpl{
```

Comment:
i would refactor the game, because it is not a interface class. Maybe GameboardStateMessage would be better